### PR TITLE
LPS-133537 The shortcut selector does not restrict previewable files to images

### PR DESCRIPTION
**Pull request by boton**
BUG: https://issues.liferay.com/browse/LPS-133537

Fixed with @AliciaGarciaGarcia

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp [hunk 1]
```diff
@@ -365,6 +365,8 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 									<%
 									FileVersion latestFileVersion = fileEntry.getLatestFileVersion();
 
+									boolean isEditable = repositoryEntryBrowserDisplayContext.isEditable(latestFileVersion);
```
Comment (adolfopa) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp:368]: By convention, boolean variable names in Java drop the "is" prefix. It should be:
```boolean editable = ...```

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by Hanlf** [comment]
The failure about portal-license-smoke-jdk8/0 is not related to this PR, rerun it,

**Comment by Hanlf** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp (hunk 1 above)
Comment (boton) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp]: Thanks, updated

**Comment by boton** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**